scripts/frida/log_converter.py

```diff
@@ -0,0 +1,74 @@
+""""
+Frida Log Converter Tool
+
+Convert Frida raw logs to standard JSON format for capa analysis
+
+Usage:
+    python3 log_converter.py <frida_log> <package_name> [output.json]
+"""
+
+import json
+import sys
+from pathlib import Path
+
+
+def convert_frida_log_to_json(log_file: Path, package_name: str, output_file: Path = None) -> Path:
+    """
+    Convert Frida log to standard JSON format
+    
+    Returns: output file path
+    """
+    api_calls = []
+    
+    with open(log_file, 'r') as f:
+        for line_num, line in enumerate(f, 1):
+            if 'FRIDA_JSON:' in line:
+                json_start = line.find('FRIDA_JSON:') + 11
+                json_str = line[json_start:].strip()
+                data = json.loads(json_str)
```

Comment:
We should be using a Pydantic model to load and validate the JSON, like you do later in the extractor.